src/main/java/ua/javaPro/berezhnoy/hillelHomeWork6/phonebook/Phonebook.java

```diff
@@ -0,0 +1,36 @@
+package ua.javaPro.berezhnoy.hillelHomeWork6.phonebook;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Phonebook {
+    static HashMap<String, ArrayList<String>> name = new HashMap<>();
+
+    public static void add(String nameKey, String number) {
+        ArrayList<String> subscriberData = new ArrayList<>();
+        if (name.containsKey(nameKey)) {
+            subscriberData = name.get(nameKey);
+            subscriberData.add(number);
+        } else {
+            subscriberData.add(number);
+            name.put(nameKey, subscriberData);
+        }
+    }
+
+    public static String find(String nameGet, int index) {
+        if (name.size() >= index) {
+            return nameGet + " " + name.get(nameGet).get(index);
+        }
+        return null;
+    }
+
+    public static String findAll(String nameGet) {
```

Comment:
```suggestion
    public static List<PhoneBookRecord> findAll(String nameGet) {
```
PhoneBookRecord (чи як ти Записа назовеш), ось вимога: 

Метод здійснює пошук записів на ім'я. Якщо **записи** знайдені, тоді **їх** необхідно повернути,